### PR TITLE
ICDS Dashboard - Beneficiary reports

### PR DESCRIPTION
**Pull request by lwyszomi**
Hi @emord,

I added changes in ```child_health_monthly_view``` as you mentioned in the case, also I moved each view to separate files. 

I checked the performance of beneficiary reports and all works quicker. Please let me know if you have any questions.

Regards,
Łukasz

### CODE REVIEW

File: custom/icds_reports/migrations/sql_templates/database_views/child_health_monthly.sql [hunk 1]
```diff
@@ -0,0 +1,113 @@
+CREATE TABLE IF NOT EXISTS public."config_report_icds-cas_static-child_health_cases_a46c129f"
```
Comment (emord) [inline custom/icds_reports/migrations/sql_templates/database_views/child_health_monthly.sql]: I think you should be able to get rid of this now correct? You may also need to remove https://github.com/dimagi/commcare-hq/blob/master/custom/icds_reports/tests/__init__.py#L97

File: custom/icds_reports/migrations/sql_templates/database_views/child_health_monthly.sql [hunk 2]
```diff
@@ -0,0 +1,113 @@
...
+        ELSE '' END AS current_month_wasting_v2,
+      CASE
+        WHEN child_health_monthly.zscore_grading_hfa_recorded_in_month = 1 THEN child_health_monthly.zscore_grading_hfa
+        ELSE 4 END AS current_month_stunting_v2_sort,
+      CASE
+        WHEN (child_health_monthly.zscore_grading_wfh = 1 AND child_health_monthly.zscore_grading_wfh_recorded_in_month = 1) OR (child_health_monthly.muac_grading = 1 AND child_health_monthly.muac_grading_recorded_in_month = 1) THEN 1
+        WHEN (child_health_monthly.zscore_grading_wfh = 2 AND child_health_monthly.zscore_grading_wfh_recorded_in_month = 1) OR (child_health_monthly.muac_grading = 2 AND child_health_monthly.muac_grading_recorded_in_month = 2) THEN 2
+        WHEN (child_health_monthly.zscore_grading_wfh = 3 AND child_health_monthly.zscore_grading_wfh_recorded_in_month = 1) OR (child_health_monthly.muac_grading = 3 AND child_health_monthly.muac_grading_recorded_in_month = 3) THEN 3
+        ELSE 4 END AS current_month_wasting_v2_sort
+  FROM "public"."awc_location_months" "awc_location_months"
```
Comment (emord) [inline custom/icds_reports/migrations/sql_templates/database_views/child_health_monthly.sql]: It looks like you moved the view files and change the definition in one commit. Could you split those up to two different commits?

Comment (lwyszomi) [reply custom/icds_reports/migrations/sql_templates/database_views/child_health_monthly.sql]: @emord I split to two commits, in the first I moved other views definition to separate files, and in the second I moved ```child_health_monthly_view``` definition to the same directory and I added changes.

**Comment by lwyszomi** [comment]
@emord Could you kick off the build? I think failing is not related.

**Comment by emord** [comment]
This looks good from me, but the aggregation script has taken quite a while since the person & mother names were added. I have a PR to make these more parallel which should help the timing. going to wait on that PR (deploying tomorrow) before merging this

**Comment by lwyszomi** [comment]
@emord 👍 please let me know when your PR will be merged, probably I will have some conflicts to resolve.

**Comment by emord** [comment]
I think I was able to resolve the merge conflicts: https://github.com/dimagi/commcare-hq/pull/20655/commits/07100d9292275908ee153824f2b121d61fbf5f90
